recipes/exiv2/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class Exiv2Conan(ConanFile):
+    name = "exiv2"
+    description = "Exiv2 is a C++ library and a command-line utility " \
+                  "to read, write, delete and modify Exif, IPTC, XMP and ICC image metadata."
+    license = "GPL-2.0"
+    topics = ("conan", "image", "exif", "xmp")
+    homepage = "https://www.exiv2.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_png": [True, False],
+        "with_xmp": [False, "bundled", "external"],
```

Comment:
Not really a fan of a using a bundled version. It allows the possibility to have 2 xmp libraries in a project.
Would making the option `[True, False]` work?

Reply:
Regarding the bundled xmp library - it is not the same as the external Adobe XMP SDK so it is not the vendoring case which we avoid in CCI. It is used by default in binary distribution of exiv2 (checked their CI).

Upstream cmake [checks](https://github.com/Exiv2/exiv2/blob/main/cmake/findDependencies.cmake#L32) for both options set so only one implementation is chosen. How we can allow using of external xmp with only two values?

Reply:
Ok, so three options are needed.
This bundled xmp library, is it only available in exiv2 or also as an external library?

How about renaming `external` to something containing `adobe`? (e.g. `adobe-xmp-toolkit`)
`requirements` will need to add a requirement to `adobe-xmp` then.
Because we don't have a recipe for https://github.com/adobe/XMP-Toolkit-SDK/, we will have to raise an error for now.

Reply:
>  This bundled xmp library, is it only available in exiv2 or also as an external library?

Looks like it is xmp-toolkit-sdk too but a very old version (4.4.0). So both of them are Adobe's xmp libraries.

Reply:
Okay then. It's not perfect but workable.

To avoid multiple xmp libraries in a project, I would conditionally add `libxmp` to `provides`.
Something like:
```python
class Exiv2ConanFile(ConanFile):
    provides = []
    def configure(self):
        if self.options.with_xmp == "bundled":
            self.provides.append("libxml")
```

If a future `adobe-xmp-toolkit` recipe would do the same, then conan can catch 2 xmp implementations in one project.

Reply:
Good idea. Added `provides` as a `xmp-toolkit-sdk` (official name of the package)